src/frontend/Sider/Personoversikt/OpprettNyBehandlingModal.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react';
+
+import { styled } from 'styled-components';
+
+import { Button, Select, VStack } from '@navikt/ds-react';
+
+import KlageGjelderTilbakekreving from './KlageGjelderTilbakekreving';
+import { useApp } from '../../context/AppContext';
+import { Feilmelding } from '../../komponenter/Feil/Feilmelding';
+import { ModalWrapper } from '../../komponenter/Modal/ModalWrapper';
+import DateInput from '../../komponenter/Skjema/DateInput';
+import { RessursStatus } from '../../typer/ressurs';
+
+export const DatoWrapper = styled.div`
+    margin-top: 1rem;
+`;
+
+interface OpprettKlageRequest {
+    mottattDato: string;
+    klageGjelderTilbakekreving: boolean;
+}
+
+const OpprettNyBehandlingModal = () => {
+    const { request } = useApp();
+
+    const [visModal, settVisModal] = useState(false);
+    const [nyBehandlingValg, settNyBehandlingValg] = useState('');
+    const [klageGjelderTilbakekreving, settKlageGjelderTilbakekreving] = useState<boolean>(false);
+    const [kravMottattDato, settKravMottattDato] = useState('');
```

Comment:
Tenker det som er koblet til behandlingstypen klage kan være i en egen i en egen komponent. Då blir statet i denne litt enklere og ikke like mye logikk.

Reply:
Eks bake inn allt i KlageGjelderTilbakekreving men at den får et nytt navn

Reply:
Har trukket ut komponentene som er spesifikke for klage-behandling til en egen komponent (OpprettKlageBehandling): 52ae4074e53fd003c4cdc0798dfac20e7ed400be

Reply:
`klageGjelderTilbakekreving` burde også kunne flyttes?

Reply:
`klageGjelderTilbakekreving` sendes inn til `ModalWrapper` på nivået over, så jeg ser ikke at den skal ligge i `OpprettKlageBehandling `

Den informasjonen må jo sendes med i POSTEN mot backend

Reply:
`const opprettKlage = (data: OpprettKlageRequest)` burde også flyttes til klage-komponenten for modalen.
Å opprette vanlig behandling eller klagebehandling er 2 ulike endepunkter